src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { PreparedTodo } from './components/types/preparedTodo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const preparedTodos: PreparedTodo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
+  };
+});
+
+let maxTodoId = [...todosFromServer].sort((el1, el2) => (
+  el2.id - el1.id))[0].id;
```

Comment:
move this to addTodo callback
and better to map => getArray of all id's => use `Math.max()`